app/styles/app.scss

```diff
@@ -1,14 +1,14 @@
 // DO NOT CHANGE:
 // load order of files below is intentional
-@import "../../bower_components/wikia-style-guide/src/scss/normalize",
-"../../bower_components/wikia-style-guide/src/scss/lib/components/global",
-"../../bower_components/wikia-style-guide/src/scss/lib/components/buttons",
-"../../bower_components/wikia-style-guide/src/scss/lib/components/dropdown",
-"../../bower_components/wikia-style-guide/src/scss/lib/components/forms",
-"../../bower_components/wikia-style-guide/src/scss/lib/components/imagery",
-"../../bower_components/wikia-style-guide/src/scss/lib/components/pagination",
-"../../bower_components/wikia-style-guide/src/scss/lib/components/type";
-@import '../../bower_components/design-system/dist/scss/index';
```

Comment:
what has changed that we don't need this `../../`?

Reply:
I pass what files to include in sassOptions in ember-cli file